src/renderers/shared/fiber/ReactFiberClassComponent.js

```diff
@@ -73,6 +74,28 @@ module.exports = function(scheduleUpdate : (fiber: Fiber, priorityLevel : Priori
     },
   };
 
+  function checkShouldComponentUpdate(workInProgress, oldProps, newProps, newState) {
+    if (oldProps === null ||
```

Comment:
formatting please:

```
if (
  oldProps === null ||
  (workInProgress.updateQueue && workInProgress.updateQueue.isForced)
) {
```

(Also, may as well pull updateQueue out to a var like before?)